### PR TITLE
Adding new wasting eqns using isanaka paper (7/9)

### PR DESCRIPTION
**Pull request by beatrixkh**

### CODE REVIEW

File: docs/source/gbd2020_models/risk_exposures/wasting_state_exposure/index.rst [hunk 1]
```diff
@@ -490,7 +490,66 @@ pool.
 Here we include equations for the transition probabilities, and in the section 
 that follows we will detail how to calculate all the variables used
 
-.. list-table:: Wasting transition probability equations
+.. list-table:: NEW WASTING TRANSITION PROBABILITY EQNS
+   :widths: 5 15 10 10
+   :header-rows: 1
+
```
Comment (rmudambi) [inline docs/source/gbd2020_models/risk_exposures/wasting_state_exposure/index.rst:496]: Do you define the `ap` terms anywhere? I don't see it, but I could be missing it.

**Review by rmudambi — APPROVED** [review]

File: docs/source/gbd2020_models/risk_exposures/wasting_state_exposure/index.rst (hunk 1 above)
Comment (beatrixkh) [reply docs/source/gbd2020_models/risk_exposures/wasting_state_exposure/index.rst:496]: yes! they're defined here: 
image

Comment (beatrixkh) [reply docs/source/gbd2020_models/risk_exposures/wasting_state_exposure/index.rst:496]: (the table that starts on line 593 @rmudambi )

File: docs/source/gbd2020_models/risk_exposures/wasting_state_exposure/index.rst [hunk 2]
```diff
@@ -490,7 +490,66 @@ pool.
...
+   * - i3
+     - ap0*f4/ap4 + ap3*r4/ap4 - d4
+     - Probability of incidence into cat 3 from cat 4
+     - System of equations
+   * - r2
+     - 0.512/60.5
+     - Probability of remission into cat 2 from cat 1 (untreated)
+     - Nicole's calculations; also referred to as r2ux (get lit source!)
+   * - r3
+     - 0.512/63 + 0.488/41.3
```
Comment (rmudambi) [inline docs/source/gbd2020_models/risk_exposures/wasting_state_exposure/index.rst:522]: Can we get these as variables, rather than numbers?

Comment (beatrixkh) [reply docs/source/gbd2020_models/risk_exposures/wasting_state_exposure/index.rst:522]: those numbers come directly from nicole's papers. i can give it to you as
r3 = treatment_coverage * time_to_treated_recovery + (1-treatment_coverge)*time_to_untreated_recovery

is that preferable / what you mean @rmudambi ?

File: docs/source/gbd2020_models/risk_exposures/wasting_state_exposure/index.rst (hunk 1 above)
Comment (Ninicorn) [reply docs/source/gbd2020_models/risk_exposures/wasting_state_exposure/index.rst:496]: To be consistent with the current wasting treatment PR, 
duration of cat 1 = 54.5 days; 
duration of cat 2 = 52.4 days (0.512*63 + 0.488*41.3)
duration of cat 3 = ? (can leave as 365 for now)

I know Abie wants to simplify things so we might revise this after discussions with Abie.

File: docs/source/gbd2020_models/risk_exposures/wasting_state_exposure/index.rst (hunk 2 above)
Comment (Ninicorn) [reply docs/source/gbd2020_models/risk_exposures/wasting_state_exposure/index.rst:522]: ah, these equations are in the wasting treatment PR! (which I will merge today)

**Review by Ninicorn — APPROVED** [review]

**Pull request merged**